dice-game/service/Cargo.lock

```diff
@@ -0,0 +1,569 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Delete this file.

Reply:
addressed in https://github.com/oasislabs/tutorials/pull/56